src/main/java/org/broadinstitute/hellbender/tools/exome/pulldown/BayesianHetPulldownCalculator.java

```diff
@@ -9,6 +9,7 @@
 import htsjdk.samtools.reference.ReferenceSequenceFileWalker;
 import htsjdk.samtools.util.IntervalList;
 import htsjdk.samtools.util.SamLocusIterator;
+import java.nio.file.Path;
```

Comment:
Unused import.

Reply:
Removed.

Reply:
It's back now, but also it's used. I was able to switch `BayesianHetPulldownCalculator` to `Path`.